release_docs/RELEASE.txt

```diff
@@ -560,6 +560,20 @@ New Features
 
     Tools:
     ------
+    - Added h5delete tool
+
+        Deleting HDF5 storage when using the VOL can be tricky when the VOL
+        does not create files. The h5delete tool is a simple wrapper around
+        the H5Fdelete() API call that uses the VOL specified in the
+        HDF5_VOL_CONNECTOR environment variable to delete a "file". If
+        the call to H5Fdelete() fails, the tool will attempt to use
+        the POSIX remove(3) call call to remove the file.
```

Comment:
I think the meaning would be clear with just "...use the POSIX remove(3) call to remove the file."

Reply:
Typo - will fix.

Reply:
Fixed